src/components/collapsible/Collapsible.js

```diff
@@ -40,7 +39,13 @@ type State = {
     isOpen: boolean,
 };
 
+type Ref = {
+    current: null | HTMLElement,
+};
+
 class Collapsible extends React.Component<Props, State> {
```

Comment:
Might be fine changing this to a React.PureComponent too, since props/state is primitives mostly